### ISSUE TITLE
Navbar stays white, even when using a dark theme 

### ISSUE THREAD

**Issue by incIined** [ccrama/Slide#3015]
Slide 6.2
LG G8 (Android 9.0, Apr security patch)

![](https://i.imgur.com/TVFml7I.jpg)

**Comment by Tensounder54** [comment]
Can we change the colour of Nav Bar? I thought that was a system thing? If we can, I'll take a look at this at some point.

**Comment by Reixko** [comment]
@Tensounder54 yes, that is actually possible, default color is main color, but I think that you can override some options of the nav bar in code, I once did something similar, so I could perhaps work on this ?

**Comment by dimitrisdimos00** [comment]
Hello, is this still an issue?

**Comment by dhruvpatidar359** [comment]
Hello , if the issue is still there then i am ready to work for it,
